core/lib/dal/src/models/storage_block.rs

```diff
@@ -43,6 +43,8 @@ pub struct StorageL1BatchHeader {
     pub bootloader_code_hash: Option<Vec<u8>>,
     pub default_aa_code_hash: Option<Vec<u8>>,
     pub protocol_version: Option<i32>,
+    pub system_logs: Vec<Vec<u8>>,
```

Comment:
add comments:
mention how it differs from l2_to_l1_logs. 
(and that it might not be set for older batches, right)?